SConstruct

```diff
@@ -86,6 +86,8 @@ env_base.android_appattributes_chunk = ""
 env_base.disabled_modules = []
 env_base.use_ptrcall = False
 env_base.split_drivers = False
+env_base.Decider('MD5-timestamp')
```

Comment:
As the options are a bit hidden here in the middle of the script, it might be worth adding a comment with a link to the documentation to explain what they're meant to do.